### PR TITLE
Clean up logging.

### PR DESCRIPTION
**Pull request by IlyaSukhanov**
Use logger named after module, a widely followed python convention.

### CODE REVIEW

File: bin/acfg1.py [hunk 1]
```diff
@@ -179,8 +181,8 @@ def get_or_create_ids(username, groupname):
     parser.add_argument("destination", help="Destination of where to overlay source directory over")
     parser.add_argument('--dry', action='store_const', const=True, default=False, help='Dry run only')
 
-    logger = logging.getLogger('')
-    logger.setLevel(logging.INFO)
+    local_logger = logging.getLogger('')
+    local_logger.setLevel(logging.INFO)
```
Comment (IlyaSukhanov) [inline bin/acfg1.py]: Just use logger here. No need to change behavior of the rest.

File: bin/disco_ssh.py [hunk 2]
```diff
@@ -57,7 +58,7 @@ def aws(self):
     def instances(self):
         """Lazily fetches all instances in the environment"""
         if not self._instances:
-            logging.info("Fetching info about instances in %s", self.env)
+            logger.info("Fetching info about instances in %s", self.env)
```
Comment (IlyaSukhanov) [inline bin/disco_ssh.py]: Maybe convert these to prints as well?

**Comment by IlyaSukhanov** [comment]
So why do this anyway? One part is convention, but another is allow easier customization of log verbosity. Right now when --debug is used we pull in all kind of events all the way to boto authenticating with AWS. It can really drown out events that we directly generate. Even with this change you'll be able to grep out all asiaq log events and that'll provide a high level overview of when something went awry.

**Comment by pchand-amplify** [comment]
isnt there a way to set the verbosity of boto3? without changing our code base this extensively?

**Comment by aoskotsky-amplify** [comment]
👍 after conflicts are resolved

**Comment by IlyaSukhanov** [comment]
@pchand-amplify yep, we already do. But we'd still not be following python logging convention. It might solve our immediate problems but if you were to use asiaq as a library you'd really have to work around the logging problems we introduced.

**Comment by pchand-amplify** [comment]
ok. 👍.

**Comment by bwarfield-amplify** [comment]
+:100: to this

**Pull request merged**